packages/server/graphql/handleGraphQLTrebuchetRequest.ts

```diff
@@ -57,16 +57,18 @@ const handleGraphQLTrebuchetRequest = async (
       const messageType = result.data ? 'complete' : 'error'
       return {type: messageType, id: opId, payload: safeResult} as const
     } catch (e) {
-      if (e?.message === 'TIMEOUT') {
-        sendToSentry(new Error('GQL executor took too long to respond'), {
-          tags: {
-            authToken: JSON.stringify(authToken),
-            docId: docId || '',
-            query: query || '',
-            variables: JSON.stringify(variables)
-          }
-        })
-        return {data: null, errors: [{message: 'The request took too long'}]}
+      if (e instanceof Error) {
+        if (e.message === 'TIMEOUT') {
```

Comment:
To avoid nested if statements I'd change the first conditional to `e instanceof Error && e.message === 'TIMEOUT'`